kolibri/plugins/learn/assets/src/views/exam-page/index.vue

```diff
@@ -110,16 +111,26 @@
         closeExam: actions.closeExam,
       },
     },
+    created() {
+      this._throttledSaveAnswer = throttle(this.saveAnswer.bind(this), 500, { leading: false });
+    },
     methods: {
       checkAnswer() {
         if (this.$refs.contentRenderer) {
           return this.$refs.contentRenderer.checkAnswer();
         }
         return null;
       },
+      throttledSaveAnswer(...args) {
+        return this._throttledSaveAnswer(...args);
+      },
       saveAnswer() {
-        const answer = this.checkAnswer();
-        if (answer && !isEqual(answer.answerState, this.currentAttempt.answer)) {
+        const answer = this.checkAnswer() || {
+          answerState: null,
+          simpleAnswer: '',
+          correct: 0,
+        };
```

Comment:
This default object on the right of the `||` could be returned by `this.checkAnswer` under the appropriate condition. It seems that the intent of this is make the type of `answer` consistent, so that it is always a `{ answerState, ..}` object instead of a `maybe { answerstate }` as it was before.